utils/buttons.py

```diff
@@ -500,35 +552,35 @@ def add_item(self, item: ui.Item) -> None:
         item.callback = partial(self.handle_callback, coro)
         super().add_item(item)
 
-    async def handle_callback(self, coro: Callable[[ui.Button, discord.Interaction], Coroutine[None, None, None]],
+    async def handle_callback(self, coro: Callable[[ui.Button, discord.Interaction], Awaitable[None]],
                               button: ui.Button, interaction: discord.Interaction, /) -> None:
         self.current_button = button
         self.current_interaction = interaction
         await coro(button, interaction)
 
-    @ui.button(emoji='<:before_fast_check:754948796139569224>', style=discord.ButtonStyle.blurple)
-    async def first_page(self, *_: Union[ui.Button, discord.Interaction]):
+    @button(emoji='<:before_fast_check:754948796139569224>', style=discord.ButtonStyle.blurple)
+    async def first_page(self, _: discord.Button, __: discord.Interaction) -> None:
```

Comment:
`discord.Button` and `ui.Button` isn't the same. dpy will always gives you a class under `discord.ui.Item` which is this case a `discord.ui.Button`. Fix this for all the other similar typing